app/scripts/data-fetchers/genbank-fetcher.js

```diff
@@ -106,18 +106,22 @@ class GBKDataFetcher {
     this.dataConfig = dataConfig;
     this.trackUid = slugid.nice();
 
+    const extension = dataConfig.url.slice(dataConfig.url.length - 3);
+    const gzipped = extension === '.gz';
+    this.errorTxt = '';
+
     this.dataPromise = fetch(dataConfig.url, {
       headers: {
-        'Content-Encoding': 'gzip'
+        'Content-Encoding': gzipped ? 'gzip' : 'text/plain',
```

Comment:
Just out of curiosity: what if a server automatically gzips the returned plain text. Would this still work?

Reply:
That should get decoded by the browser and be returned as plaintext to fetch.